dynatrace/main.py

```diff
@@ -57,3 +58,4 @@ def __init__(
         self.third_part_synthetic_tests: ThirdPartySyntheticTestsService = ThirdPartySyntheticTestsService(self.__http_client)
         self.tokens: TokenService = TokenService(self.__http_client)
         self.maintenance_windows = MaintenanceWindowService(self.__http_client)
+        self.metric_events = MetricEventService(self.__http_client)
```

Comment:
@jimm-with-a-j  Only one request, let's call this `self.anomaly_detection_metric_events`, I think it will make more sense for users and when we add the rest of the `anomaly_detection` endpoints as well